### ISSUE TITLE
developers.useflashpunk.net not working anymore :(

### ISSUE THREAD

**Issue by enbyspiders** [useflashpunk/useflashpunk.github.io#3]
For a while now the forums seem to be gone.

In Firefox I just get a blank page.
Testing in Chrome I get an HTTP ERROR 500.

I thought this might just have been some hosting issues but I think it's been more than a month now, maybe?
Anyone still there?

**Comment by zachwlewis** [comment]
Yeah, it's been down for a spell. I'm not sure what it is. I think it outgrew the space I'm using on Digital Ocean. I either need to jump in and purge some data or upgrade to a bigger setup. I'll look into it.

**Comment by zachwlewis** [comment]
@voec So. Good news and bad news. The good news is I was able to update the forum to the latest version and get the mail service replaced (we had to switch from Mandrill to SparkPost because Mail Chimp made Mandrill part of their subscription plan). The bad news is... none of the data migrated over during the update, so the forum is basically a default install.

I've got multiple backups of the old install since before it went down, and I've set the forum to read-only mode to prevent people trying to post or sign up. I spent the past few days getting the forum to this point. Now, I've got to figure out how to restore the old data. Once I do that, everything should be back to the way it was before it went down.

So, that's the current status. I'll post any updates here as they roll in.
